docs/content/doc/help/faq.en-us.md

```diff
@@ -118,6 +118,11 @@ The correct path for the template(s) will be relative to the `CustomPath`
 2. If you are still unable to find a path, the default can be [calculated above](#where-does-gitea-store-what-file)
 3. Once you have figured out the correct custom path, you can refer to the [customizing Gitea]({{< relref "doc/administration/customizing-gitea.en-us.md" >}}) page to add your template to the correct location.
 
+## Does Gitea have a "GitHub/GitLab pages" feature?
+
+Gitea will not implement this feature due to CSRF security concerns.
```

Comment:
Hmm, not sure we need to call it out as a bloat feature necessarily.

Reply:
Ah yes. I have no idea why I called it that :wink::
![image](https://user-images.githubusercontent.com/51889757/231018722-cf320945-f7ca-4abf-8fe9-d5954e7a5b03.png)

Reply:
Size is relative, and many of those deps would probably just be part of what we already have.

Either way, I don't think it needs to be a specific reference against it, just that there's already an optional app that does it and integrates well with Gitea.
(Just imo)

Reply:
```suggestion
Gitea doesn't provide builtin Pages server. You need a dedicated domain to serve static pages to avoid CSRF security risks.

For simple usage, you can use a reverse proxy to rewrite & serve static contents from Gitea's raw file URLs.

```